### PR TITLE
Enable support for 3scale/operator support for ppc64le(tested on UBI9)

### PR DESCRIPTION
**Pull request by shubham-bhagwat0**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by shubham-bhagwat0** [comment]
[scri-logs.txt](https://github.com/ppc64le/build-scripts/files/15041549/scri-logs.txt)
tested on UBI 9.3

File: 3/3scale-operator/3scale-operator_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+PACKAGE_NAME=3scale-operator
+PACKAGE_VERSION=master
+PACKAGE_URL=https://github.com/3scale/3scale-operator
+PACKAGE_BRANCH=master
+
+#Install the required dependencies
+yum install git gcc make wget tar zip -y
+
+git clone $PACKAGE_URL
```
Comment (AbhiIBM) [inline 3/3scale-operator/3scale-operator_ubi_9.3.sh:29]: Cloning package twice

File: 3/3scale-operator/3scale-operator_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package	    : 3scale-operator
+# Version	    : master
+# Source repo	: https://github.com/3scale/3scale-operator
+# Tested on	    : UBI 9.3
```
Comment (AbhiIBM) [inline 3/3scale-operator/3scale-operator_ubi_9.3.sh:7]: Tested on value should be "UBI:9.3" else the script will fail.

**Review by shubham-bhagwat0 — COMMENTED** [review]
changes made, fixed suggested issues

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**